src/postgres/commands/registerPostgresCommands.ts

```diff
@@ -15,7 +20,16 @@ import { deletePostgresTable } from "./deletePostgresTable";
 import { enterPostgresCredentials } from "./enterPostgresCredentials";
 import { openPostgresFunction } from "./openPostgresFunction";
 
+export const connectedDBKey: string = 'ms-azuretools.vscode-azuredatabases.connectedPostgresDB';
```

Comment:
I think this should be named `connectedPostgresKey` and go back and rename `connectedDBKey` in `registerMongoCommands.ts` to `connectedMongoKey`.

Having them both be called connectedDBKey is confusing when you're importing them into files.